src/app.html

```diff
@@ -10,7 +10,7 @@
     <link rel="icon" type="image/png" sizes="16x16" href="%sveltekit.assets%/favicon-16x16.png" />
     <link rel="manifest" href="%sveltekit.assets%/site.webmanifest" />
     <meta name="msapplication-TileColor" content="hsl(0, 0%, 100%)" />
-    <meta name="theme-color" content="hsl(220, 66%, 21%)" />
+    <meta name="theme-color" content="#122a59" />
```

Comment:
let uses hsl, not sure why you change this?

Reply:
because when I was checking this it didn't work hsl(220, 66%, 21%) it came out blank and I decided to change to the other format and it worked

Reply:
when you pushed it first time it worked. hsl will work, so please fix the real issue rather than doing weird stuff